index.ts

```diff
@@ -225,6 +244,45 @@ async function getAllIssues(ownerName: string, repoName: string) {
   return issues;
 }
 
+/**
+ * Returns all org repositories urls or owner/repo url
+ * @param orgOrRepo org or repository name
+ * @returns array of repository urls
+ */
+async function getRepoUrls(orgOrRepo: string) {
+  const params = orgOrRepo.split("/");
+  let repos: string[] = [];
+  switch (params.length) {
```

Comment:
I think its unusal to use switch case in this context but given that the codebase is small and simple this should not be an issue.